### ISSUE TITLE
cannot checkout on windows due to invalid chars in filename

### ISSUE THREAD

**Issue by mitchcapper** [cloudflare/cfssl#589]
The addition of certain filenames prevents cfssl from being able to checkout on windows any more:

```
git clone https://github.com/cloudflare/cfssl.git cftest
Cloning into 'cftest'...
remote: Counting objects: 6855, done.
remote: Compressing objects: 100% (71/71), done.
Receiving objects: 100% (6855/6855), 8.10 MiB | 423.00 KiB/s, done.1

Resolving deltas: 100% (2913/2913), done.
Checking connectivity... done.
error: unable to create file vendor/github.com/cloudflare/cfssl_trust/ca-bundle/SwissSignCA(RSAIKMay6199918:00:58)_2000-11-26_SHA1WithRSA.crt (Invalid argument)
error: unable to create file vendor/github.com/cloudflare/cfssl_trust/ca-bundle/SwissSignCA(RSAIKMay6199918:00:58)_2000-11-26_SHA1WithRSA_2.crt (Invalid argument)
Checking out files: 100% (3122/3122), done.
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry the checkout with 'git checkout -f HEAD'
```

I can submit a PR if desired but I am not sure if just renaming them is the best solution or what have you.

**Comment by kisom** [comment]
This should be changed in [cfssl_trust](https://github.com/cloudflare/cfssl_trust) and the vendored version updated in CFSSL. A rename of the current certs works, but it's also probably useful to add a change to [format.go](https://github.com/cloudflare/cfssl_trust/blob/master/cmd/format/format.go) to Windows-sanitise paths.

**Comment by mitchcapper** [comment]
This will be fixed when cloudflare/cfssl_trust#23 is finally done.
